### ISSUE TITLE
Pěkná ikonka do hlášky o neveřejnosti v Krameriu

### ISSUE THREAD

**Issue by LenkaKravecova** [moravianlibrary/Rozcestnik-studovna#43]
Dát pěknou ikonku do Krameria do hlášky o neveřejnosti dokumentu (pouze v budově MZK - rightMsgAlternative).
Něco takového:
![kramerius-pacho](https://cloud.githubusercontent.com/assets/11834597/15112041/dd16a1fe-15eb-11e6-8178-ed3ca222216e.png)

**Comment by honza-rychtar** [comment]
Bude potreba vytvorit hlasky dve - Martin rikal, ze to jde a ze to tak udelate.
1) pro pristup z nechraneneho prohlizece z pocitacu v MZK - tato hlaska by mela uzivatele nasmerovat do chaneneho prohlizece. Podobne jako na screenshotu, ale klidne vic popisne: "Na plose klikne te na ikonu ..." a obrazek te ikony.
2) pro pristup mimo MZK - tady by se spis melo vysvetlit, proc neni dokument verejny, na koho se obratit, pokud si myslim, ze by dokument mel byt verejny a upozornit na moznost pristupu z budovy MZK. Tady uz ale neuvadet podrobny navod o chranenem prohlizeci.

**Comment by LenkaKravecova** [comment]
Jo, přesně tak už to je udělané ;) Jde jen o přidání pěkné ikonky, která se bude zobrazovat v hlášce v budově MZK, teď je tam jen obyč. odkaz.
